src/libaktualizr/primary/aktualizr_test.cc

```diff
@@ -422,11 +422,15 @@ void process_events_FullMultipleSecondaries(const std::shared_ptr<event::BaseEve
     allcomplete_FullMultipleSecondaries = true;
   } else if (event->variant == "PutManifestComplete") {
     manifest_FullMultipleSecondaries = true;
+  } else if (event->variant == "DownloadPaused") {
+    // phony event on shutdown, ignore
+    return;
   }
   // It is possible for the PutManifestComplete to come before we get the
   // InstallTargetComplete depending on the threading, so check for both.
   if (allcomplete_FullMultipleSecondaries && complete_FullMultipleSecondaries == 2 &&
       manifest_FullMultipleSecondaries) {
+    LOG_ERROR << "SETVALUE";
```

Comment:
Is this leftover from testing or intentional?

Reply:
@patrickvacek That's  debug output, thanks, I'll remove it.